packages/core/src/interpreter.ts

```diff
@@ -654,9 +661,11 @@ export class Interpreter<
       this.parent && (to === SpecialTargets.Parent || this.parent.id === to);
     const target = isParent
       ? this.parent
-      : isActor(to)
+      : isString(to)
+      ? this.children.get(to as string) || registry.get(to as string)
+      : isActor(to) || isSpawnedActor(to)
```

Comment:
There is no way for `isSpawnedActor` returning true without `isActor` returning true, so this check is redundant
```suggestion
: isActor(to)
```